test/unit/controllers/actionbuilders/AuthActionSpec.scala

```diff
@@ -123,7 +123,7 @@ class AuthActionSpec extends UnitSpec with MockitoSugar {
   }
 
   "NonCspAuthAction" should {
-    "Authorise Non CSP when authorised by auth API " in new SetUp {
+    "Return Right of AuthorisedRequest with AuthorisedAs Non CSP when authorised by auth API " in new SetUp {
```

Comment:
I think for test specs, it is better to use the functional language i.e. Authorise instead of Return Right or Return Left

Reply:
As this is at a unit level, the wording is technical and describes the function / expected function of the code being tested.

Reply:
I have changed reference to `Left` and `Right` to  use the functional test language